From 8a9c6862c6010b4500cb08c62ac4aea6897cf485 Mon Sep 17 00:00:00 2001
From: Eric Anholt <eric@anholt.net>
Date: Fri, 9 Mar 2018 14:24:05 -0800
Subject: [PATCH] ARM: BCM270X: Add the 18-bit DPI pinmux to the RPI
 DTs.

This doesn't do anything by default, but trying to put the node in an
overlay failed for me.

Signed-off-by: Eric Anholt <eric@anholt.net>
---
 arch/arm/boot/dts/bcm270x.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

--- a/arch/arm/boot/dts/bcm270x.dtsi
+++ b/arch/arm/boot/dts/bcm270x.dtsi
@@ -19,6 +19,13 @@
 
 		gpio@7e200000 { /* gpio */
 			interrupts = <2 17>, <2 18>;
+
+			dpi_18bit_gpio0: dpi_18bit_gpio0 {
+				brcm,pins = <0 1 2 3 4 5 6 7 8 9 10 11
+					     12 13 14 15 16 17 18 19
+					     20 21>;
+				brcm,function = <BCM2835_FSEL_ALT2>;
+			};
 		};
 
 		serial@7e201000 { /* uart0 */
